include/paths.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * paths.h compatibility shim
+ */
+
+#ifdef HAVE_PATHS_H
+
+#include_next <paths.h>
+
+#else
+
+#ifndef LIBCOMPAT_PATHS_H
+#define LIBCOMPAT_PATHS_H
+
+#define    _PATH_DEVNULL   "/dev/null"
```

Comment:
Works for me. I can't imagine many if any systems targeted by openntpd would have /dev/null someplace else ;).